Description: Disable failing run-make test on armhf
 Last checked this still affects 1.18
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/issues/43329
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/test/run-make-fulldeps/extern-fn-struct-passing-abi/Makefile
+++ b/src/test/run-make-fulldeps/extern-fn-struct-passing-abi/Makefile
@@ -1,5 +1,7 @@
 -include ../tools.mk
 
 all: $(call NATIVE_STATICLIB,test)
+ifeq (,$(filter armv7-unknown-linux-gnueabihf,$(TARGET)))
 	$(RUSTC) test.rs
 	$(call RUN,test) || exit 1
+endif
